readthedocs/donate/templates/donate/ethicalads.html

```diff
@@ -0,0 +1,91 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block title %}{% trans "Pay for your ad" %}{% endblock %}
+
+{% block extra_links %}
+  <link rel="stylesheet" href="{% static 'payments/css/form.css' %}" />
+{% endblock %}
+
+{% block extra_scripts %}
+  <script src="https://js.stripe.com/v2/" type="text/javascript"></script>
+  <script type="text/javascript" src="{% static 'vendor/knockout.js' %}"></script>
+  <script type="text/javascript" src="{% static 'donate/js/donate.js' %}"></script>
+  <script type="text/javascript">
+var donate_views = require('donate/donate');
+$(document).ready(function () {
+    var key;
+    //<![CDATA[
+    key = '{{ stripe_publishable }}';
+    //]]>
+
+    var view = donate_views.DonateView.init({
+      key: key,
+      form: $('form#donate-payment')
+    });
+});
+  </script>
+{% endblock %}
+
+
+{% block content %}
+  <h2>Pay for your Sponsorship</h2>
+
+  <p>
+    This form can be used to pay for your sponsorship of Read the Docs.
+    Thanks for helping make Open Source more sustainable!
+  </p>
```

Comment:
Same with the title and this paragraph block. Framing this as sponsorship is misleading.